sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -68,11 +68,17 @@ internal CustomFormModel(Model model, FormRecognizerClientOptions.ServiceVersion
         /// <summary>
         /// An optional, user-defined name to associate with your model.
         /// </summary>
+        /// <remarks>
+        /// This property only has value for <see cref="FormRecognizerClientOptions.ServiceVersion.V2_1_Preview_3"/> and up.
```

Comment:
Would it make sense to change this to `ServiceVersion.V2_1` since we won't have another release with preview.3 now?

Reply:
I could. I was thinking of waiting to target the v2.1 GA swagger and then update the `ServiceVersion` enum so it only has `v2.0` and `v2.1` values. The good thing is that when I do that, all this docstrings will be automatically updated as they are treated as a reference and not as a string